<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Teacher</class>
 <widget class="QWidget" name="Teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_course1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>course1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_course2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>course2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_course3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>course3</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>611</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>29</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>320</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STHeiti</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>欢迎您：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>课程平均分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Aver_Course">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
